src/Pim/Bundle/ReferenceDataBundle/Doctrine/ORM/Repository/ReferenceDataRepository.php

```diff
@@ -90,6 +90,18 @@ public function findOneByIdentifier($identifier)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function findByIdentifiers(array $referenceDataCodes)
```

Comment:
It was a discussion we had with @jjanvier, we choose to fetch full entities because it could be reused. But yes, actually, I would agree with you that it is not the purpose yet. :)